eo-maven-plugin/src/test/java/org/eolang/maven/MjResolveIT.java

```diff
@@ -47,6 +47,27 @@ void resolvesJarFile(@Mktmp final Path temp) throws IOException {
         );
     }
 
+    @Test
+    void logsJarWithEmptyClassifierCorrectly(@Mktmp final Path temp) throws IOException {
+        final String version = "0.52.0";
```

Comment:
@h1alexbel I think the build will fail right after new release with new version because of this line

Reply:
@maxonfjvipon maybe its better to don't test logging? WDYT?